src/Thread.ts

```diff
@@ -72,4 +72,8 @@ export default class Thread {
 	compact() {
 		this.trace.compact();
 	}
+
+	clear() {
+		(this.trace as any) = null;
```

Comment:
Should the trace property on this class be nullable now that we may assign null? This would save us the casting here.

Reply:
I decided not to do this as the property should only be null for inactive threads in the oldThreadsList. Making the type nullable would require dealing with the null possibility (type-wise) at every access.